tests/test_requestcache.py

```diff
@@ -1,6 +1,9 @@
+# THIS MUST BE IMPORTED EARLIER APPARENTLY, COSTED TONS OF TIME TO FIGURE OUT.
+# IF THIS IS NOT FIRST, THE REACTOR WILL NOT BE RUNNING AND EVERYTHING WILL CRASH AND BURN.
```

Comment:
This is no longer true, I moved the reactor import stuff to __init__.py to avoid having to worry about these things. 

Just rebase and undo this change and everything will be fine.